spec/factories.rb

```diff
@@ -1,2 +1,8 @@
 FactoryGirl.define do
```

Comment:
Probably better to use Nathan's factory here since it will pass his validations (in particular...belonging to company).